understanding/accessible-authentication-enhanced.html

```diff
@@ -53,136 +53,115 @@
+                  <p><a href="#dfn-cognitive-function-test">認知機能テスト</a> (パスワードを記憶する、パズルを解く、など) は、認証<a href="#dfn-process">プロセス</a>のどのステップにおいても要求されない。ただし、そのステップが次の少なくとも一つを提供する場合は除く。
                   </p>
                   <dl>
                      
-                     <dt>Alternative</dt>
+                     <dt>代替手段</dt>
                      
-                     <dd>Another authentication method that does not rely on a cognitive function test.</dd>
+                     <dd>認知機能テストに依存しない別の認証方法がある。</dd>
                      
-                     <dt>Mechanism</dt>
+                     <dt>メカニズム</dt>
                      
-                     <dd>A <a href="#dfn-mechanism">mechanism</a> is available to assist the user in completing the cognitive function test.
+                     <dd>認知機能テストを利用者が完遂できるように支援する<a href="#dfn-mechanism">メカニズム</a>が利用できる。
                      </dd>
                      
                   </dl>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>意図</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make logins possible with less mental effort.</dd>
+                  <dt>目標</dt>
+                  <dd>より少ない精神的労力でログインできるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Don't make people recognize objects or user-supplied images and media to login.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ログインするために、人々に物体又は利用者が提供した画像及びメディアを認識させない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with cognitive disabilities can't do puzzles, including identifying objects
-                     and non-text information they previously supplied.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>認知障害のある人々の中には、以前に提供した物体及び非テキストの情報を特定するなどの、パズルができない人もいる。
                   </dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use,
-                  and secure method to log in, access content, and undertake tasks. This criterion is
-                  the same as <a href="accessible-authentication-minimum">Accessible Authentication (Minimum)</a> but without the exceptions for objects and user-provided content.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ログインし、コンテンツにアクセスし、タスクを実行するための、アクセシブルで、使いやすく、安全な方法を確保することである。この基準は、<a href="accessible-authentication-minimum">アクセシブルな認証 (最低限)</a> と同じであるが、物体及び利用者提供のコンテンツの例外はない。
                </p>
-               <p>Any required step of the authentication process:</p>
+               <p>全ての認証プロセスの必要なステップ:</p>
```

Comment:
この箇条書きは、この達成基準 3.3.9 においては、認められないということが言いたいのだと思いますので、以下、いかがでしょうか？

```suggestion
               <p>認証プロセスに必要なあらゆるステップにおいては:</p>
```